src/ray/protobuf/export_event.proto

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020-2021 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+option cc_enable_arenas = true;
+
+package ray.rpc;
+
+import "google/protobuf/struct.proto";
+
+message ExportEvent {
+  enum SourceType { EXPORT_TASK = 0; }
+
+  // event_id is the unique ID of this event
+  string event_id = 1;
+  // source type is the type of the source
+  SourceType source_type = 2;
+  // source_hostname is the hostname of the source
+  // timestamp is the report milliseconds since 00:00, Jan 1 1970 UTC
+  int64 timestamp = 8;
+  // event_data follows the schema associated with the source_type
+  google.protobuf.Struct event_data = 9;
+}
```

Comment:
If we think TaskEvents is the public spec, we should move it here to make it clear that it's part of the public contract.

Instead of `google.protobuf.Struct`, we should do something like 

```
oneof event_data {
   TaskEvents task;
}
```